api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -200,11 +200,10 @@ default AppendFiles newFastAppend() {
   ExpireSnapshots expireSnapshots();
 
   /**
-   * Create a new {@link Rollback rollback API} to roll back to a previous snapshot and commit.
-   *
-   * @return a new {@link Rollback}
+   * Create a new {@link ManageSnapshots API} to manage snapshots in this table and commit.
+   * @return
    */
-  Rollback rollback();
```

Comment:
Can we deprecate this instead to allow users a chance to migrate? We could build an implementation of Rollback that translates to the `ManageSnapshots` API for now.

Reply:
done